pkg/kp/rollstore/consul_store.go

```diff
@@ -476,6 +480,13 @@ func (s consulStore) CreateRollingUpdateFromOneMaybeExistingWithLabelSelector(
 	if err != nil {
 		return roll_fields.Update{}, err
 	}
+	if createdOld {
```

Comment:
The reason for delaying label creating for the new RC is "Now that we know there are no RUs in progress, and we have the update creation locks, we can safely apply labels.". Does the same reasoning apply to the old RC as well? If it does not, the labels can be created earlier. I will attempt to reason about this question.